python/ambassador_diag/diagd.py

```diff
@@ -957,6 +961,7 @@ def show_overview(reqid=None):
     # to compute -- we don't want to call it more than once here, so we cache
     # its value.
     diag = app.diag
+    assert diag
```

Comment:
In python, does any of these assertions crash Emissary? Are we adding them as a way to detect bugs?